### DISCUSSION TITLE
Implementation of features described in papers

### DISCUSSION THREAD

**Discussion by dooglewoogle** [pystiche/pystiche#592] [Ideas]
I've been working on a proof of concept implementation of ways to control stylisation factors described [here](https://arxiv.org/abs/1606.05897) and expanded upon [here](https://arxiv.org/abs/1611.07865)
Would you be interested in me tidying it up and sending a pull request? 

Also, would this be better suited for pystiche/pystiche or pystiche/papers? I ask because it would be implementation of a paper. However the second paper I linked describes a method spacial control. I don't know if it's mathmatically similar, but it's functionally identical to guided transformations in pystiche as it uses black/white masking images.

Cheers.

**Comment by pmeier** [comment]
> I've been working on a proof of concept implementation of ways to control stylisation factors described [here](https://arxiv.org/abs/1606.05897) and expanded upon [here](https://arxiv.org/abs/1611.07865)
> Would you be interested in me tidying it up and sending a pull request?

If you are talking about the color preservation, I think we already have part of this covered [here](https://github.com/pystiche/papers/blob/a614f4effdfcbf3d3fe45202151fdc14c610e2b3/replication/gatys_et_al_2017/main.py#L125).
